### ISSUE TITLE
Append also values of StartLine, Wait for Start, End Copper to the querystring

### ISSUE THREAD

**Issue by KONEY** [grahambates/gradient-blaster#6]
In order to recreate the exact same copperlist at reload values from said fields should be added in the generated querystring.

I've been wondering if there was any reason for them not to be included but so far my only good explanation is they were forgotten :)

**Comment by grahambates** [comment]
Yeah, in general it doesn't export output settings at the moment, but I agree it would be good if it did.
